modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsManagerImpl.java

```diff
@@ -429,6 +430,10 @@ private void checkStatisticsSupport(String op) throws IgniteCheckedException {
            throw new IgniteCheckedException(String.format(
                "Unable to perform %s due to not all server nodes supports STATISTICS_COLLECTION feature.", op));
        }
+
+       if (ctx.state().clusterState().state() != ClusterState.ACTIVE)
+           throw new IgniteException(String.format(
+               "Unable to perform %s due to cluster state [state=%s]", op, ctx.state().clusterState().state()));
```

Comment:
Why not MessageFormat.format?

Reply:
String.format used more than 100 times in codebase while MessageFormat.format - only 9. Is there are some advantages in your method?